uMenu/source/ui/ui_Actions.cpp

```diff
@@ -43,10 +45,12 @@ namespace ui::actions {
         std::string name;
         os::GetAccountName(uid, name);
         const auto option = g_MenuApplication->CreateShowDialog(GetLanguageString("user_settings"), GetLanguageString("user_selected") + ": " + name + "\n" + GetLanguageString("user_option"), { GetLanguageString("user_view_page"), GetLanguageString("user_logoff"), GetLanguageString("cancel") }, true, os::GetIconCacheImagePath(uid));
+        ui::quickmenu_utils::quickMenuInputsToIgnore=1;
         if(option == 0) {
             friendsLaShowMyProfileForHomeMenu(uid);
         }
         else if(option == 1) {
+            ui::quickmenu_utils::quickMenuInputsToIgnore=0; //Menu can close if i am logging out
```

Comment:
` = ` on assignments (same for the rest of the changes, although not that important)

Reply:
no problems, i will change it now according to your style :)

Reply:
sorry i did not understood that